doc/src/build/install.md

```diff
@@ -2,24 +2,55 @@
 title: Install Sui
 ---
 
-Sui is written in Rust, and we are using Cargo to build and manage the
-dependencies.  As a prerequisite, you will need to [install
-Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html)
-version 1.60.0 or higher in order to build and install Sui on your machine.
-Also ensure you have `cmake` installed:
-```shell
-$ which cmake
-```
-
-> **Tip:** Assuming you have Rust Cargo, the `git` command, and a GitHub account,
-> you can download the `sui-setup.sh` script and run it to conduct all of the setup
-> below, from a terminal like so:
+Welcome to the Sui development environment! Get started immediately:
+
+1. Meet the [prerequisites](#prerequisites).
+2. Install the [binaries](#binaries).
+3. Configure an [Integrated Development Environment (IDE)](#integrated-development-environment).
+4. Optionally, download the [source code](#source-code) to have local
+   access to examples and modify Sui itself.
+
+> **Tip:** Assuming you have Rust Cargo, the `git` command, and a GitHub account
+> (see prerequisites(#prerequisites)), you can download the `sui-setup.sh` script
+> and run it to conduct all of the setup below, **including removal of any existing
+> sui assets**. To use it, run these commands in a terminal:
 > ```shell
 > $ curl https://raw.githubusercontent.com/MystenLabs/sui/main/doc/utils/sui-setup.sh -o sui-setup.sh
 > chmod 755 sui-setup.sh
 > ./sui-setup.sh
 > ```
 
+## Prerequisites
+
+At a minimum, you should have a machine capable of installing command line tools.
+These prerequisites are broken down into the [essential](#essential) tools
+you need to work in Sui and the [advanced](#advanced) items needed for Sui source
+code development.
+
+### Essential
+
+Sui is written in Rust, and we are using Cargo to build and manage the
+dependencies. You will need Cargo to build and install Sui on your machine.
+
+To run Sui, you will need:
+1. A command line interface, as virtually everything done here is done by CLI.
+1. The `curl` command to download other tools, which you can confirm with:
+   ```shell
+   $ which curl
+1. The [Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) toolchain version 1.60.0 or higher; update it with:
+   ```shell
+   $ rustup update stable
+   ```
+1. The `cmake` command installed.
+1. The Sui [binaries](#binaries).
```

Comment:
We're building from source: how does that require binaries?

Reply:
Hrm, I got the impression from @awelc the binaries are always needed. How would you run Wallet?

And this section at the top merely summarizes the steps documented lower in the page. If I remove it, aren't we indicating to get started they no longer need the binaries? I'm confused...

Reply:
Let me put this differently: if we use action verbs, such as
1. "You need to install the `cmake` command"
2. "you need to compile and install the Sui binaries" there no longer is any confusion.

Reply:
I already introduced the list with "you will need" so that would have been redundant. I've now appended "to install" to that intro. We haven't yet said anything about compiling the binaries, but I welcome commits and PRs.

As you may recall, there is a whole lot of room for improvement there...
https://github.com/MystenLabs/sui/issues/1733